WBooks/Library/LibraryController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  LibraryController.swift
+//  WBooks
+//
+//  Created by Nacho Mendez on 04/08/2021.
+//
+
+import UIKit
+
+class LibraryController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    private lazy var library: LibraryView = LibraryView()
+    
+    override func loadView() {
+        view = library
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        library.tableBooks.delegate = self
+        library.tableBooks.dataSource = self
+        
+        let nib = UINib.init(nibName: "MyCustomCell", bundle: nil)
```

Comment:
`init` keyword is not necessary. Also, try to use more expressive names like `LibraryCell` instead of `MyCustomCell`.